BrandonBeltran/hackerrank/04/JavaCurrencyFormatter.java

```diff
@@ -0,0 +1,23 @@
+package bootcamp;
+
+
+import java.text.NumberFormat;
+import java.util.Locale;
+import java.util.Scanner;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+
+public class JavaCurrencyFormatter {
+    private static final Logger logger = LogManager.getLogger(Test.class);
```

Comment:
If this variable is static final, what is the correct name? change in all the code